pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1937,16 +1958,16 @@ class Patch(CommonObject, object):
     application : :class:`pyaedt.hfss.Hfss`
         HFSS design or project where the variable is to be created.
     frequency : float, None
-        The patch frequency, it is used in prediction formulas. If it is None, the patch frequency will be that of the
+        The target resonant frequency for the patch antenna. If it is None, the patch frequency will be that of the
```

Comment:
```suggestion
        Target resonant frequency for the patch antenna. The default is ``None``,
        in which case the patch frequency is that of the
```
Present tense. I see the subsequent parameter descriptions aren't following our guidelines.